templates/All/Default.html

```diff
@@ -1,44 +1,36 @@
 <div class="coveo-result-frame">
-  <div class="coveo-result-row">
-    <div class="coveo-result-cell" style="width:85px;text-align:center;padding-top:7px;">
-      <span class="CoveoIcon">
-      </span>
-      <div class="CoveoQuickview">
+  <div class="coveo-result-cell" style="vertical-align:top;text-align:center;width:32px;">
```

Comment:
Not _everything_ in classes :)

What I meant was for common things. Some examples: 

- `coveo-result-row` with `margin-top:10px`, 
- `CoveoFieldValue` with `margin-right:30px;`
- `coveo-result-cell` with `vertical-align:top;text-align:center;width:32px;`

Some value I see for classes:

- They are things you might _all_ want to change one-shot instead of individually changing them. 
- They could have names, which would make them easier for the reader. For example, the third one could have a `filetype-section` or something.
- It is way easier to override classes than inline styles. You simply have to define a higher priority class instead of patching with `!important` or something like that.

I am fine with leaving it as is, even more if it is for the Interface Editor. Just wanted to raise a consideration :)